curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-375-minimum-of-subsequences.md

```diff
@@ -8,43 +8,38 @@ dashedName: problem-375-minimum-of-subsequences
 
 # --description--
 
-Let Sn be an integer sequence produced with the following pseudo-random number generator:
+Let $S_n$ be an integer sequence produced with the following pseudo-random number generator:
 
-S0
+$$\begin{align}
+        S_0 & = 290\\,797 \\\\
+  S_{n + 1} & = {S_n}^2\bmod 50\\,515\\,093
```

Comment:
The `{S_n}^2` doesn't quite look the same. [Euler site](https://projecteuler.net/problem=375) is on the left, this PR is on the right:
![Screen Shot 2021-07-29 at 8 01 38 AM](https://user-images.githubusercontent.com/20648924/127496788-a2ff069b-04c4-478d-8a65-2a5fe49fa284.png)
I feel like we've had a discussion around this before - not sure if they mean the same thing.

Reply:
I think previously it was the other way around.

I believe both are meaning basically the same - second power of S_n. They look different, because they are differently written `S_n^2` vs `{S_n}^2`. Euler's page don't have this consistent, as this seems to be the first time where the first option is used.

Reply:
Sounds good to me.